features/unhandled_signal.feature

```diff
@@ -7,13 +7,22 @@ Feature: Signals are captured as error reports in Bugsnag
     When I run "AbortScenario" and relaunch the app
     And I configure Bugsnag for "AbortScenario"
     And I wait to receive an error
-    Then the error is valid for the error reporting API version "4.0" for the "iOS Bugsnag Notifier" notifier
+    Then the error is valid for the error reporting API
     And the error payload field "events" is an array with 1 elements
     And the exception "errorClass" equals "SIGABRT"
+    # On ARM the stack looks like:
+    #   __pthread_kill
+    #   pthread_kill
+    #   abort
+    #   -[AbortScenario run]
+    #   ...
+    #
+    # On Intel, "pthread_kill" does not appear in the stack trace:
+    #   __pthread_kill
+    #   abort
+    #   -[AbortScenario run]
+    #   ...
```

Comment:
Follow on ticket?

Reply:
@nickdowell Is there an existing ticket to address this?  I can create one otherwise.

Reply:
I don't this there is a resolution to this, other than maybe not pedantically checking the stack traces in scenarios?

Reply:
👉 PLAT-5861